src/dlgTriggerEditor.cpp

```diff
@@ -8392,6 +8392,7 @@ void dlgTriggerEditor::changeView(EditorViewType view)
     treeWidget_variables->setVisible(view == EditorViewType::cmVarsView);
     checkBox_displayAllVariables->setVisible(view == EditorViewType::cmVarsView);
 
+    mpToggleActiveAction->setEnabled(view != EditorViewType::cmVarsView);
```

Comment:
We might want to consider making the test for more of these to be `(view != EditorViewType::cmVarsView) && (view != EditorViewType::cmUnknownView)` given that we seem to be hitting odd cases of the latter value initially...